src/sema/unused_variable.rs

```diff
@@ -487,3 +487,26 @@ pub fn check_unused_events(ns: &mut Namespace) {
         }
     }
 }
+
+/// Check for unused errors
```

Comment:
Sorry about that :rofl: BUT while there are missing comments on other functions, this comment seems overly superficial. I'd either remove or add more context than what's already obvious from the function name (given this is a public function, giving more context would be nice).

Reply:
Absolutely right, I've added a much better comment